sites/partners/pages/application/[id]/review.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useMemo, useState, useContext } from "react"
+import Head from "next/head"
+import dayjs from "dayjs"
+import { useSWRConfig } from "swr"
+import { useRouter } from "next/router"
+import { GridApi } from "ag-grid-community"
+import { useForm } from "react-hook-form"
+import {
+  t,
+  Button,
+  NavigationHeader,
+  AlertBox,
+  AppearanceStyleType,
+  useMutate,
+  StatusBar,
+  AgTable,
+  useAgTable,
+  GridSection,
+  Modal,
+  AppearanceBorderType,
+  Field,
+} from "@bloom-housing/ui-components"
+import { useSingleFlaggedApplication } from "../../../lib/hooks"
+import Layout from "../../../layouts"
+import { getCols } from "./applicationsCols"
+import { AuthContext } from "@bloom-housing/shared-helpers"
+import {
+  ApplicationFlaggedSet,
+  ApplicationReviewStatus,
+  EnumApplicationFlaggedSetStatus,
+  EnumApplicationFlaggedSetResolveStatus,
+  ApplicationFlaggedSetResolve,
+} from "@bloom-housing/backend-core/types"
+
+const Flag = () => {
+  const router = useRouter()
+  const flagsetId = router.query.id as string
+
+  const [saveModalOpen, setSaveModalOpen] = useState(false)
+  const [gridApi, setGridApi] = useState<GridApi | null>(null)
+
+  const columns = useMemo(() => getCols(), [])
+
+  const { data, cacheKey } = useSingleFlaggedApplication(flagsetId)
+  const { reset, isSuccess, isLoading, isError } = useMutate<ApplicationFlaggedSet>()
+  const { applicationFlaggedSetsService } = useContext(AuthContext)
+
+  const { mutate } = useSWRConfig()
+
+  const { mutate: saveSetMutate, isLoading: isSaveLoading } = useMutate()
+
+  const saveSet = (formattedData: ApplicationFlaggedSetResolve) => {
+    void saveSetMutate(() =>
+      applicationFlaggedSetsService
+        .resolve({
+          body: formattedData,
+        })
+        .then(() => {
+          // next issue: set success alert
```

Comment:
Is this issue created already? If so could you put it in this comment so we can better track it

Reply:
Sure thing! #2860